Edgex-Vault fails to starting listener with binding address already in use

When docker-compose up is run, the edgex-vault container currently fails to start the listener with the following error messages: 

```
==> Vault server configuration:

             Api Address: https://edgex-vault:8200
                     Cgo: disabled
         Cluster Address: https://edgex-vault:8201
              Listener 1: tcp (addr: "127.0.0.1:8200", cluster address: "edgex-vault:8201", max_request_duration: "1m30s", max_request_size: "33554432", tls: "enabled")
              Listener 2: tcp (addr: "edgex-vault:8200", cluster address: "edgex-vault:8201", max_request_duration: "1m30s", max_request_size: "33554432", tls: "enabled")
               Log Level: info
                   Mlock: supported: true, enabled: true
                 Storage: consul (HA available)
                 Version: Vault v1.0.2
             Version Sha: 37a1dc9c477c1c68c022d2084550f25bf20cac33

==> Vault server started! Log data will stream in below:

2019-10-24T22:31:16.261Z [INFO]  core: security barrier not initialized
2019-10-24T22:31:16.323Z [INFO]  core: security barrier not initialized
2019-10-24T22:31:16.325Z [INFO]  core: security barrier initialized: shares=5 threshold=3
2019-10-24T22:31:16.346Z [INFO]  core: post-unseal setup starting
2019-10-24T22:31:16.356Z [INFO]  core: loaded wrapping token key
2019-10-24T22:31:16.356Z [INFO]  core: successfully setup plugin catalog: plugin-directory=
2019-10-24T22:31:16.356Z [INFO]  core: no mounts; adding default mount table
2019-10-24T22:31:16.359Z [INFO]  core: successfully mounted backend: type=kv path=secret/
2019-10-24T22:31:16.359Z [INFO]  core: successfully mounted backend: type=cubbyhole path=cubbyhole/
2019-10-24T22:31:16.359Z [INFO]  core: successfully mounted backend: type=system path=sys/
2019-10-24T22:31:16.359Z [INFO]  core: successfully mounted backend: type=identity path=identity/
2019-10-24T22:31:16.364Z [INFO]  core: successfully enabled credential backend: type=token path=token/
2019-10-24T22:31:16.364Z [INFO]  core: restoring leases
2019-10-24T22:31:16.364Z [INFO]  rollback: starting rollback manager
2019-10-24T22:31:16.364Z [INFO]  expiration: lease restore complete
2019-10-24T22:31:16.369Z [INFO]  identity: entities restored
2019-10-24T22:31:16.371Z [INFO]  identity: groups restored
2019-10-24T22:31:16.372Z [INFO]  core: post-unseal setup complete
2019-10-24T22:31:16.372Z [INFO]  core: starting listener: listener_address=192.168.80.7:8201
2019-10-24T22:31:16.372Z [INFO]  core: serving cluster requests: cluster_listen_address=192.168.80.7:8201
2019-10-24T22:31:16.372Z [INFO]  core: starting listener: listener_address=192.168.80.7:8201
2019-10-24T22:31:16.372Z [ERROR] core: error starting listener: error="listen tcp 192.168.80.7:8201: bind: address already in use"
```

--- a/releases/nightly-build/compose-files/docker-compose-nexus.yml
+++ b/releases/nightly-build/compose-files/docker-compose-nexus.yml
@@ -90,36 +90,11 @@ services:
       - "IPC_LOCK"
     environment:
       - VAULT_ADDR=https://edgex-vault:8200
-      - |
-        VAULT_LOCAL_CONFIG=
-          listener "tcp" { 
-              tls_disable = "0" 
-              cluster_address = "edgex-vault:8201" 
-              tls_min_version = "tls12" 
-              tls_client_ca_file ="/run/edgex/secrets/edgex-vault/ca.pem" 
-              tls_cert_file ="/run/edgex/secrets/edgex-vault/server.crt" 
-              tls_key_file = "/run/edgex/secrets/edgex-vault/server.key" 
-              tls_perfer_server_cipher_suites = true 
-          } 
-          backend "consul" { 
-              path = "vault/" 
-              address = "edgex-core-consul:8500" 
-              scheme = "http" 
-              redirect_addr = "https://edgex-vault:8200" 
-              cluster_addr = "https://edgex-vault:8201" 
-          } 
-          default_lease_ttl = "168h" 
-          max_lease_ttl = "720h"
       - VAULT_CONFIG_DIR=/vault/config
       - VAULT_UI=true
-    # create the tempfs directory inside the container for deployed purpose
-    tmpfs:
-      - /run/edgex/secrets/edgex-vault
     command: >
       /usr/bin/dumb-init -- /bin/sh -c 
-      "./security-secrets-setup generate ;
-      chown -Rh 100:1000 /run/edgex/secrets/edgex-vault/ ;
-      exec /usr/local/bin/docker-entrypoint.sh server -log-level=info"
+      "exec /usr/local/bin/docker-entrypoint.sh server -log-level=info"
     volumes:
       - vault-file:/vault/file
       - vault-logs:/vault/logs

